src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh

```diff
@@ -41,7 +41,7 @@ on_error() {
 trap 'on_error $? $LINENO' ERR
 trap 'cleanup $? $LINENO' EXIT
 
-sed -i '/ceph-node-/d' $HOME/.ssh/known_hosts
+sed -i '/ceph-node-/d' $HOME/.ssh/known_hosts || true
```

Comment:
Nothing against grouping related changes in a PR, but it's not a good practice to mask bugfixes behind improvements/cleanups.

It seems to me that the core issue in this PR is this line, which is a bugfix, and everything else is an improvement. Am I right? So I'd change the title of the PR/commit with a "fix..." (the upgrade would be secondary, wouldn't it?).

Reply:
you are right. changed the commit and pr title